### ISSUE TITLE
explain error

### ISSUE THREAD

**Issue by moodymudskipper** [moodymudskipper/ask#6]
From error message, trace, and constructing the inputs (with a cap on object size, where we'd replace with a summary).

I think we need `context_error()` that calls `context_stack()` and `context_env()` called on every env of the stack.

**Comment by krlmlr** [comment]
Is there a way to access the model's output if it can't be parsed by, e.g., `ask_in_place()` ?

**Comment by moodymudskipper** [comment]
yes even for functions that do post processing such as `ask_in_place()` the unprocessed output should be available in the `conversation_manager()`
